### ISSUE TITLE
Cannot enable Jupyter extension when connected to remote machine with ssh

### ISSUE THREAD

**Issue by amunger** [microsoft/vscode#153171]
Issue Type: <b>Bug</b>

1. Connect to a remote machine with the ssh extension
2. install the jupyter extension on the remote machine
3. disable the extension, reload the window, then re-enable

Result:
Error: `Cannot change enablement of Jupyter Notebook Renderers extension because of its extension kind`
The extension has to be uninstalled and reinstalled to get it to load again

VS Code version: Code - Insiders 1.69.0-insider (173f4bea73601a31859372b409cac2c20fa91194, 2022-06-24T06:32:08.615Z)
OS version: Windows_NT x64 10.0.22000
Restricted Mode: No
Remote OS version: Linux x64 5.13.0-1021-azure

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-1065G7 CPU @ 1.30GHz (8 x 1498)|
|GPU Status|2d_canvas: enabled<br>canvas_oop_rasterization: disabled_off<br>direct_rendering_display_compositor: disabled_off_ok<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>opengl: enabled_on<br>rasterization: enabled<br>raw_draw: disabled_off_ok<br>skia_renderer: enabled_on<br>video_decode: enabled<br>video_encode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|31.59GB (11.66GB free)|
|Process Argv|--folder-uri file:///c%3A/src/test/3 --crash-reporter-id 1202dc88-403d-4ebc-8393-af3551a8bd5b|
|Screen Reader|no|
|VM|0%|

|Item|Value|
|---|---|
|Remote|SSH: 20.57.139.129|
|OS|Linux x64 5.13.0-1021-azure|
|CPUs|Intel(R) Xeon(R) CPU E5-2673 v3 @ 2.40GHz (1 x 2394)|
|Memory (System)|3.34GB (0.19GB free)|
|VM|0%|
</details><details><summary>Extensions (10)</summary>

Extension|Author (truncated)|Version
---|---|---
Bookmarks|ale|13.3.0
tsl-problem-matcher|amo|0.6.2
vscode-language-pack-de|MS-|1.69.6220922
jupyter-keymap|ms-|1.0.0
remote-containers|ms-|0.240.0
remote-ssh|ms-|0.83.2022062315
remote-wsl|ms-|0.66.3
jupyter|ms-|2022.6.1001771053
jupyter-keymap|ms-|1.0.0
jupyter-renderers|ms-|1.0.8

</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
vstes627:30244334
vslsvsres303:30308271
pythonvspyl392:30422396
pythontb:30258533
pythonptprofiler:30281269
vshan820:30294714
pythondataviewer:30285072
vscod805cf:30301675
bridge0708:30335490
bridge0723:30353136
vsaa593cf:30376535
pythonvs932:30404738
wslgetstarted:30449409
vscscmwlcmt:30465136
cppdebug:30492333
pylanb8912:30496796
vsclangdf:30492506
841h4636:30516487

```

</details>

**Comment by sunnycarter** [comment]
Hi there, I raised a duplicate issue and was asked if I would try "Insiders" to see if it reproed. 
https://github.com/microsoft/vscode/issues/154147#issuecomment-1176514536 

I installed Insiders and it did not repro so seems fixed. What I don't understand is whether I should keep Insiders installed, or uninstall my normal VSCode and use it instead. I use VSCode on WSL.  There was no explanation of what to do with Insiders.
I am going on the assumption that I will uninstall it and wait for this to make it into a mainline VSCode update.

**Comment by sandy081** [comment]
> I am going on the assumption that I will uninstall it and wait for this to make it into a mainline VSCode update.

Right and this fix is available in VS Code (stable) now.

**Comment by mlsad3** [comment]
I don't think this is actually fixed. Maybe it started working for @sunnycarter due to doing a reinstall (switching to Insiders)?
 - I did a brand new install of Code (first time on this machine) yesterday
 - Version 1.69.1, Date: 2022-07-12 (2 days ago)
 - Installed ssh extension and connected remote
 - Source code did not have any Git change information (no line markers for lines that have changed)
 - https://stackoverflow.com/a/70296872/277601 recommended disabling built-in 'Git' and re-enabling it
 - Same issue happened: "Cannot change enablement of Git Base extension because of its extension kind"

Strange workaround:
 - Click 'Enable for Workspace'
 - Then click 'Enable'
 - It appears to be enabled (although the source code still does not show git change information...but that's a different issue)

**Comment by sandy081** [comment]
Can you please provide a gif to get better understanding of what is happening?

**Comment by mlsad3** [comment]
Here are the steps in the video:
 - Open remote connection using ssh extension
 - Go to extensions, filter by "Built In"
 - Select Git, and Disable
 - Reload
 - Go to extensions, select Git
 - Try Enabling (fails)
 - Go to Explorer, open a workspace
 - Go to extensions, select Git
 - Try Enabling (fails)
 - Enable for workspace (success)
 - Try Enabling (success)

![git_enable_disable](https://user-images.githubusercontent.com/15711477/179269813-42d139c2-e782-4396-a11a-0d27b96f7333.gif)
